packages/codegen-ui/lib/types/form/form-metadata.ts

```diff
@@ -16,6 +16,11 @@
 import { DataFieldDataType } from '../data';
 import { FieldValidationConfiguration } from './form-validation';
 
+/**
+ * Form Action type definition
+ */
+export type StudioFormActionType = 'create' | 'update';
```

Comment:
nit: i think we might want to keep this with the main form type

Reply:
Yeah, I moved it to avoid circular imports. The main form type should be moved out of index.ts